### PR TITLE
fix(tabs): border on all the width

### PR DESCRIPTION
**Pull request by aesteves60**

### CODE REVIEW

File: packages/ods/src/components/tabs/src/components/ods-tabs/ods-tabs.scss [hunk 1]
```diff
@@ -5,7 +5,15 @@
 
   display: inline-flex;
   flex-flow: row;
-  padding: var(--ods-outline-width);
-  width: inherit;
-  overflow-x: auto;
+  padding: var(--ods-outline-width) 0;
+  width: 100%;
+  overflow-y: auto;
```
Comment (dpellier) [inline packages/ods/src/components/tabs/src/components/ods-tabs/ods-tabs.scss:10]: `overflow-x` we want an horizontal scroll (it won't impact your tab rendering with the wrapper)

File: packages/ods/src/components/tabs/src/components/ods-tab/ods-tab.scss [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 :host(.ods-tab) {
   display: block;
+  margin-bottom: -2px;
```
Comment (dpellier) [inline packages/ods/src/components/tabs/src/components/ods-tab/ods-tab.scss:6]: please use a shared variable for `margin-bottom` and `border-bottom` size here and `border-bottom` size in ods-tabs.scss

File: packages/ods/src/components/tabs/src/components/ods-tabs/ods-tabs.scss [hunk 3]
```diff
@@ -5,7 +5,15 @@
...
+}
+
+.ods-tabs {
+  &-border {
```
Comment (dpellier) [inline packages/ods/src/components/tabs/src/components/ods-tabs/ods-tabs.scss:14]: - use `__` for BEM block
- use a more generic name, as this block contains the tabs, it's not just for the border (something like `__items` or `__wrapper` or `__container`)

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by dpellier — APPROVED** [review]

**Review by feoche — CHANGES_REQUESTED** [review]
## Global 
- [ ] When focusing first tab, outline is being cropped on the left part

**Review by astagnol — APPROVED** [review]

**Review by feoche — APPROVED** [review]

**Pull request merged**